test/test_managercli.py

```diff
@@ -1435,6 +1435,60 @@ def test_service_level_creates_syspurpose_dir_and_file(self):
         self.assertEqual(contents.get("service_level_agreement"), "JRJAR")
 
 
+class TestUsageCommand(TestCliProxyCommand):
+    command_class = managercli.UsageCommand
+
+    def setUp(self):
+        TestCliProxyCommand.setUp(self)
+        self.cc.consumerIdentity = StubConsumerIdentity
+        self.cc.cp = StubUEP()
+        syspurposelib.USER_SYSPURPOSE = self.write_tempfile("{}").name
+
+    def tearDown(self):
+        super(TestUsageCommand, self).tearDown()
+        syspurposelib.USER_SYSPURPOSE = "/etc/rhsm/syspurpose/syspurpose.json"
+
+    def test_main_server_url(self):
+        server_url = "https://subscription.rhsm.redhat.com/subscription"
+        self.cc.main(["--serverurl", server_url])
+
+    def test_insecure(self):
+        self.cc.main(["--insecure"])
+
+    def test_usage_not_supported(self):
+        self.cc.cp.setConsumer({})
+        try:
+            self.cc.set_usage('JARJAR')
+        except SystemExit as e:
+            self.assertEqual(e.code, os.EX_UNAVAILABLE)
```

Comment:
Use [assertRaises](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertRaises) to test exceptions